data/signin.js

```diff
@@ -0,0 +1,265 @@
+class Signin{
+
+    constructor() {
+        this.selector_type = 'dusk';
+
+        this.params = {
+            group:{
+                count: null,
+                name: null,
+            }
+        }
+
+        this.selectors = {
+            email: 'signin-email_or_username',
+            password: 'signin-password',
+            submit: 'signin-signin',
+            page_heading: 'h3',
+            home_page_heading: 'h2',
+            page_subheading: 'p',
+            alert_box: 'alertdialog',
+            menu_item: 'menuitem',
+            logout_button: '=Logout',
+            eye_icon: 'eye',
+            forgot_password: '=Forgot Password?',
+            link_text: '=VaahCMS'
+        }
+
+        this.params.page = {
+            id: "SL",
+            name: "Sign In",
+            url: '/'
+        }
+
+        this.groups = [
+            {
+                count: 1,
+                name:"Validation",
+                tests: [
+                    {
+                        count: 1.1,
+                        name: "Invalid Credentials",
+                        expect: "User should not be able to signin and an error message should be displayed",
+                        email: "sanu@gmail.com",
+                        password: "12345",
+                        assert: {
+                            error_msg: "No user exist",
+                            sign_in_page_title: "Sign In",
```

Comment:
@satyajeet-s001 why we need this `sign_in_page_title` in all `tests`?

Reply:
sir i have removed the unnecessary sign_in_page_title from the scripts , and for the groups[1], i am expecting two validations i.e.  user should not be able to signin and some error message to be displayed, so for verifying user is in signin page only after entering the wrong credentials i have asserted this as you have suggested before.